docs/foundry/getting-started.md

```diff
@@ -87,232 +88,53 @@ forge-std/=lib/forge-std/src/
-
-To create an Anvil node in a stand-alone process, run:
-
+Or generate a Javascript ChugSplash project:
 ```
-anvil
+npx chugsplash init --js
 ```
 
-In another terminal window, run the following command to deploy your upgradeable contract:
+You'll see we've created a few new files:
+- `src/HelloChugSplash.sol`: A sample contract to be deployed
+- `chugsplash/HelloChugSplash.config.ts`: A ChugSplash config file
+- `script/HelloChugSplash.s.sol`: A deployment script for deploying the sample project onto a live network.
+- `test/HelloChugSplash.t.sol`: A test script for running local tests of the project using ChugSplash.
 
-```
-forge script --rpc-url http://localhost:8545 script/MyFirstProject.s.sol
-```
+By default, the ChugSplash config for the sample project just deploys a single immutable contract HelloChugSplash. We'll explain the details of the ChugSplash config file in the next guide.
 
-You should see the following output:
-```
-== Logs ==
-  Success!
-  HelloChugSplash: 0x...
+## 6. Run the tests
+In your project root, run the tests with the following command:
 ```
-
-You've deployed your first upgradeable contract locally!
-
-## 9. Test with ChugSplash
-
-In the folder that contains your Foundry tests (usually `test/`), create your test file. We'll call it `MyFirstProject.t.sol`.
-
-Inside, copy and paste the following:
-```sol
-// SPDX-License-Identifier: MIT
-pragma solidity ^0.8.15;
-
-import "forge-std/Test.sol";
-import "chugsplash/ChugSplash.sol";
-import "../src/HelloChugSplash.sol";
-
-contract ChugSplashTest is Test {
-    // Your upgradeable contract
-    HelloChugSplash helloChugSplash;
-
-    function setUp() public {
-        // Create a ChugSplash instance
-        ChugSplash chugsplash = new ChugSplash();
-
-        // Define the path from the project root to your ChugSplash config file.
-        string memory chugsplashFilePath = "./chugsplash/hello-chugsplash.json";
-
-        // Deploy all contracts in your ChugSplash config file (in this case, just HelloChugSplash.sol)
-        chugsplash.deploy(chugsplashFilePath, true);
-
-        // You *must* refresh EVM state after calling `chugsplash.deploy`.
-        chugsplash.refresh();
-
-        // Connect to the deployed contract
-        helloChugSplash = HelloChugSplash(chugsplash.getAddress(chugsplashFilePath, "HelloChugSplash"));
-    }
-
-    function testNumber() public {
-        assertEq(helloChugSplash.number(), 1);
-    }
-
-    function testStored() public {
-        assertEq(helloChugSplash.stored(), true);
-    }
-
-    function testStorageName() public {
-        assertEq(helloChugSplash.storageName(), "First");
-    }
-
-    function testOtherStorage() public {
-        assertEq(helloChugSplash.otherStorage(), 0x1111111111111111111111111111111111111111);
-    }
-}
+forge test
 ```
 
-Notice that you need to call `chugsplash.refresh()` after calling `chugsplash.deploy(...)`.
-
-Run your tests using the command:
+## 7. Deploy on anvil
```

Comment:
IMO we should just deploy locally here via `forge script ...`. Then, we can skip the .env stuff and the need to start up a new anvil node.

Reply:
IMO the value of this step is to demonstrate the real deployment process including actually broadcasting the transactions so that it's clear how it works when broadcasting and so that this process can be replicated on a real network. I don't think there's much value in having this step at all if we aren't actually doing the broadcast. 

Also I realized I forgot to include a step on generating the deployment artifacts which is important IMO and will only work if the transactions are actually broadcast.